### ISSUE TITLE
Issue with utf8 files

### ISSUE THREAD

**Issue by mwe** [palantir/tslint#336]
Hi,

I have issues that are related to UTF8 typescript files.

The file in UTF8:

```javascript
/**
 * @file filename
 */ 
```

will be validated as // tslint : (jsdoc-format) client/tslinterror.ts[2, 1]: asterisks in jsdoc must be aligned.

```javascript
/**
  * @file filename
  */ 
```
is validated without errors (incorrect)

Any suggestions for a correct fix?  see issue (#299) .

Please find attached debug info.  startPosition is 1 incase of UTF8.

![debug-info](https://cloud.githubusercontent.com/assets/94524/6370656/0b6cc32e-bcf3-11e4-8bd8-77e6995bfed9.PNG)

**Comment by adidahiya** [comment]
@mwe is this still an issue on the latest tslint?
